src/Input.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+
+function Input({ description, onChange, onAdd }) {
+  return (
+    <p>
+      <input
+        type='text'
+        placeholder='할 일을 입력해 주세요'
+        value={description}
+        onChange={(e) => onChange(e)} >
```

Comment:
```suggestion
        onChange={onChange} >
```
input의 경우 특정한 파라미터가 없을 경우에 기본 파라미터로 이벤트 객체가 생성되기 때문에 위와 같이 처리해줘도 될 것 같아요.